docs/source/basic_concepts.rst

```diff
@@ -121,7 +121,7 @@ of each attribute):
       env.PYTHONPATH.append("{root}/python")
       env.PATH.append("{root}/bin")
 
-The :attr:`requires` section defines the requirements of the package. The :func:`commands` section describes
+The :attr:`requires` section defines the requirements of the package, excluding *any* subsequent patch versions. The :func:`commands` section describes
```

Comment:
I feel like this could be added to the "Package Requests" section. And in theory it's already covered by the `foo-1.2+<2` example, isn't it?

Reply:
I had to go back and look at the issue (#1824) to understand what this was talking about.

Maybe instead it might be worth it to add an example with the `<=` operator as in the issue, as I can see how it could be ambiguous, and is not listed in https://rez.readthedocs.io/en/stable/basic_concepts.html#package-requests